packages/components/src/components/relative-datepicker/relative-datepicker.scss

```diff
@@ -0,0 +1,18 @@
+#dc-relative-datepicker {
+    width: 100%;
+    text-align: center;
+    padding-bottom: 16px;
+
+    span {
```

Comment:
span is too general to be in the component, it is better to add a class name for this tag to follow BEM.